packages/near-fast-auth-signer/package.json

```diff
@@ -50,6 +49,7 @@
     "https-browserify": "^1.0.0",
     "i18next": "^23.2.8",
     "js-sha256": "^0.10.1",
+    "lodash": "^4.17.21",
```

Comment:
instead of loading whole lib, lets just import debounce only
https://www.npmjs.com/package/lodash.debounce

Reply:
Good point, let me change it!